lighthouse-core/gather/gatherers/offline.js

```diff
@@ -34,4 +36,8 @@ class Offline extends Gatherer {
   }
 }
 
+function cleanHash(url) {
+  return url.href.removeURLFragment();
```

Comment:
To be honest autocomplete let me to it :p. (https://cs.chromium.org/chromium/src/third_party/WebKit/Source/devtools/front_end/platform/utilities.js?q=removeURLFragment&sq=package:chromium&l=215)

I can just copy that function instead so we are sure it never disappears